pkg/tools/base/gobin.go

```
@@ -0,0 +1,114 @@
+	gobin := strings.TrimSpace(string(out))
+	if gobin != "" {
+		return gobin, nil
+	}
+
+	out, err = exec.Command(goBin, "env", "GOPATH").Output()
+	if err != nil {
+		return "", fmt.Errorf("failed to run 'go env GOPATH': %w", err)
+	}
+	gopath := strings.TrimSpace(string(out))
+	if gopath == "" {
+		return "", fmt.Errorf("both GOBIN and GOPATH are empty; cannot determine where 'go install' placed the binary")
```

golangci-lint (2.11.4)

[error] error-format: fmt.Errorf can be replaced with errors.New

(perfsprint)